include/GoogleSync/GoogleSync.php

```diff
@@ -763,11 +935,10 @@ public function delEvent($event, $meeting_id)
      *
      * @return Meeting SuiteCRM Meeting Bean
```

Comment:
function could pass false as failure. Please update the phpdoc

Reply:
Done